### PR TITLE
fix(learn): Change Node project descriptions to all use same template

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

In an effort to standardize all the Node back-end projects, I believe I fixed all relevant files.  The only ones I did not change, will be fixed (via change requests or are already present) on the following PRs:
- #40219 (Anonymous Message Board)
- #40218 (Stock Price Checker)
- #39965 (Exercise Tracker)

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/file-metadata-microservice.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ forumTopicId: 301506
 <section id='description'>
 Build a full stack JavaScript app that is functionally similar to this: <a href='https://file-metadata-microservice.freecodecamp.rocks/' target='_blank'>https://file-metadata-microservice.freecodecamp.rocks/</a>. Working on this project will involve you writing your code using one of the following methods:
 
-- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-filemetadata/'>this GitHub repo</a> and complete your project locally.
+- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-filemetadata/' target='_blank`>this GitHub repo</a> and complete your project locally.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/file-metadata-microservice.md:13]:
```suggestion
- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-filemetadata/' target='_blank'>this GitHub repo</a> and complete your project locally.
```

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Build a full stack JavaScript app that is functionally similar to this: <a href=
 
 Working on this project will involve you writing your code using one of the following methods:
 
-- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-issuetracker/'>this GitHub repo</a> and complete your project locally.
+- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-issuetracker/' target='_blank'>this GitHub repo</a> and complete your project locally.
 - Use <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-issuetracker">this repl.it starter project</a> to complete your project.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md:15]:
```suggestion
- Use <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-issuetracker" target='_blank'>this repl.it starter project</a> to complete your project.
```

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md [hunk 3]
```diff
@@ -8,10 +8,11 @@ forumTopicId: 301571
 ## Description
 <section id='description'>
 
-Build a full stack JavaScript app that is functionally similar to [this site](https://personal-library.freecodecamp.rocks/). Working on this project will involve you writing your code using one of the following methods:
+Build a full stack JavaScript app that is functionally similar to this: <a href='https://personal-library.freecodecamp.rocks/' target='_blank'></a>.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md:11]:
```suggestion
Build a full stack JavaScript app that is functionally similar to this: <a href='https://personal-library.freecodecamp.rocks/' target='_blank'>https://personal-library.freecodecamp.rocks/</a>.
```

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md [hunk 4]
```diff
@@ -8,10 +8,11 @@ forumTopicId: 301571
...
+Working on this project will involve you writing your code using one of the following methods:
 
-- Clone [this GitHub repo](https://github.com/freeCodeCamp/boilerplate-personal-library) and complete your project locally.
-- Use [our repl.it starter project](https://repl.it/github/freeCodeCamp/boilerplate-personal-library) to complete your project.
+- Clone <a href='https://github.com/freeCodeCamp/boilerplate-personal-library' target='_blank'>this GitHub repo</a> and complete your project locally.
+- Use <a href='https://repl.it/github/freeCodeCamp/boilerplate-personal-library' target='_blank'>our repl.it starter project</a>) to complete your project.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md:15]:
```suggestion
- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-library' target='_blank'>this GitHub repo</a> and complete your project locally.
- Use <a href='https://repl.it/github/freeCodeCamp/boilerplate-project-library' target='_blank'>our repl.it starter project</a>) to complete your project.
```
These links are different.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md:15]: What is the difference here?

Comment (RandellDawson) [reply curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md:15]: Never mind.  I see now.

**Comment by nhcarrigan** [comment]
Any idea why adding the example project link errors out the CI? That's the same format we used on all of the other project pages...

**Comment by RandellDawson** [comment]
@ojeytonwilliams We are getting a linting error:

```
/home/runner/work/freeCodeCamp/freeCodeCamp/curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md: 11: MD034/no-bare-urls Bare URL used [Context: "https://personal-library.freec..."]
```
The link referenced here does not seem to be a bare URL.  Why is the linter complaining?

**Comment by ojeytonwilliams** [comment]
> Why is the linter complaining?

It doesn't like bare links in Markdown, but it only interprets text as being Markdown if it's in a paragraph. You can either use `[]()` syntax or delete the empty line between <section> and the text.

`_blank` isn't great from a UX and an a11y perspective, imo, and the Markdown syntax is cleaner, so I'd prefer the former.

**Comment by RandellDawson** [comment]
> _blank isn't great from a UX and an a11y perspective, imo, and the Markdown syntax is cleaner, so I'd prefer the former.

Opening a new window is the desired effect, to avoid losing the page you were on.  We now have the user stories and instructions for the Node projects and challenges on `/learn` and just the code in the boilerplates, so it makes total sense to me to not lose the page you started on (the one with the instructions) while opening a new window with the boilerplate code.  Markdown links `[]()` do not support opening new windows, so we can not use them here. 

To help with the accessibility, we could try to implement [one of these suggested ideas](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/#how-to-inform-your-users).  In the section below the one I linked, I am reminded we should also be adding `rel="noopener"` to the links.  In the section below it, there is another section on using JavaScript.

**Comment by ojeytonwilliams** [comment]
I know there are ways to mitigate the downsides of tabs that automatically open, but I'm not a fan.  We can add 'opens in a new tab', but that means people have to read/listen to extra stuff.  Ctrl-clicking is easy.  Anyway, pet-peeve of mine and a bit off topic, so I'll move on.

> On a side note, why is the linter all of the sudden complaining about these kinds of links we have been adding throughout the site for a long time?

It happens when people include an empty line after a section, something that generally isn't done.  Hence the need for https://github.com/freeCodeCamp/freeCodeCamp/pull/37912

**Review by naomi-lgbt — DISMISSED** [review]
LGTM~! 🥳 I do think exploring alternatives to `_blank` is a worthwhile discussion, but something that can be done as a separate issue. Shipping this PR will bring our project pages to a consistent format, which is also important.

Thank you for your pull request! We look forward to additional contributions!

**Comment by ojeytonwilliams** [comment]
> LGTM~!  I do think exploring alternatives to `_blank` is a worthwhile discussion, but something that can be done as a separate issue. 

Yep, I shouldn't have brought it up here - my bad.

File: curriculum/challenges/english/10-coding-interview-prep/take-home-projects/build-an-image-search-abstraction-layer.md [hunk 5]
```diff
@@ -7,13 +7,12 @@ forumTopicId: 302361
...
 <section id='description'>
-<strong>Objective:</strong> Build a full stack JavaScript app on <a href='https://repl.it/' target='_blank'>Repl.it</a> that allows you to search for images like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10</a> and browse recent search queries like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/</a>.
+Build a full stack JavaScript app on <a href='https://repl.it/' target='_blank'>Repl.it</a> that allows you to search for images like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10</a> and browse recent search queries like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/</a>. Use a site builder of your choice to complete the project.
```
Comment (Sky020) [inline curriculum/challenges/english/10-coding-interview-prep/take-home-projects/build-an-image-search-abstraction-layer.md:10]:
```suggestion
Build a full stack JavaScript app that allows you to search for images like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10</a> and browse recent search queries like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/</a>. Use a site builder of your choice to complete the project.
```

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md [hunk 6]
```diff
@@ -9,8 +9,13 @@ forumTopicId: 301509
 
 <section id='description'>
 Build a full stack JavaScript app that is functionally similar to this: <a href='https://url-shortener-microservice.freecodecamp.rocks/' target='_blank'>https://url-shortener-microservice.freecodecamp.rocks/</a>.
-Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.
-Start this project on Repl.it using <a href='https://repl.it/github/freeCodeCamp/boilerplate-project-urlshortener' target='_blank'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-urlshortener/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+Working on this project will involve you writing your code using one of the following methods:
```
Comment (Sky020) [inline curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md:12]: For consistency, I would have `Working on this...` be on the same line as `...rocks/</a>.` before it

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md [hunk 7]
```diff
@@ -11,8 +11,8 @@ Build a full stack JavaScript app that is functionally similar to this: <a href=
 
 Working on this project will involve you writing your code using one of the following methods:
```
Comment (Sky020) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/issue-tracker.md:12]: See above ⬆️

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md [hunk 8]
```diff
@@ -8,8 +8,13 @@ forumTopicId: 301570
 ## Description
 <section id='description'>
 Build a full stack JavaScript app that is functionally similar to this: <a href="https://metric-imperial-converter.freecodecamp.rocks/" target="_blank">https://metric-imperial-converter.freecodecamp.rocks/</a>.
-Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.
-Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-metricimpconverter">this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-metricimpconverter/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+Working on this project will involve you writing your code using one of the following methods:
```
Comment (Sky020) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md:11]: See above :arrow_up:

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md [hunk 9]
```diff
@@ -7,15 +7,14 @@ forumTopicId: 301571
 
 ## Description
 <section id='description'>
+Build a full stack JavaScript app that is functionally similar to this: <a href='https://personal-library.freecodecamp.rocks/' target='_blank'>https://personal-library.freecodecamp.rocks/</a>.
+Working on this project will involve you writing your code using one of the following methods:
```
Comment (Sky020) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md:11]:
```suggestion
Build a full stack JavaScript app that is functionally similar to this: <a href='https://personal-library.freecodecamp.rocks/' target='_blank'>https://personal-library.freecodecamp.rocks/</a>. Working on this project will involve you writing your code using one of the following methods:
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
One small nitpick about some of `Working on this project...` being on its own line, and others not.

Also, conflicting information about `Use Repl.it...` and `Use a site builder of your choice...`

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md (hunk 6 above)
Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md:12]: Can you make the suggestion here on GitHub, so I can commit it or is that not possible?

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md (hunk 8 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md:11]: Can you make the suggestion here on GitHub, so I can commit it or is that not possible?

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md (hunk 6 above)
Comment (nhcarrigan) [reply curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md:12]: It looks like that's not possible - GitHub disables the suggestion feature if the selection includes deleted lines.

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md (hunk 8 above)
Comment (Sky020) [reply curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md:11]: @RandellDawson Sorry, I would have, but GitHub does not allow suggestions across deleted sections. Unless, am I missing something?

**Comment by RandellDawson** [comment]
@Sky020 I just pushed another commit to fix the two issues you found.  I also fixed the Sudoku project as it was missing the sentence entirely.

**Comment by Sky020** [comment]
@RandellDawson  The `metric-imperial-converter` still appears the same.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 Thanks, for that, Randell.

@nhcarrigan Nich, feel free to merge, if you are still happy

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Thank you for your pull request! We look forward to additional contributions!

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md (hunk 8 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md]: Done

**Pull request merged**